thorlcr/thorutil/thmem.cpp

```diff
@@ -262,12 +262,15 @@ class CSharedSpillableRowSet : public CSpillableStreamBase, implements IInterfac
             }
             else if (pos == owner->rows.numCommitted())
             {
-                owner->clearSpillingCallback();
```

Comment:
Just a question, was calling clearSpillingCallback() wrong before, since owner->spillFile is not set ?